bucket/yfanyi.json

```diff
@@ -0,0 +1,18 @@
+{
+    "version": "10.1.5",
```

Comment:
When there's no `autoupdate` defined, the version is better to be 'nightly' instead of a specific version number, for it (`nightly`) can suppress the hash checking procedure when installing the app.

Reply:
And hashes are not easy to be well synchronized with the latest upstream binary without a proper `autoupdate` .